CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ option(RE2C_REGEN_BENCHMARKS "Regenerate C code for benchmarks" OFF)
 
 # checks for programs
 find_package(BISON)
+find_package(Python3 COMPONENTS Interpreter)
```

Comment:
@sergeyklay To clarify, for now we do not require python and do not bother with the checks (if it was found) either.